### ISSUE TITLE
pods not showing for apps on catalog apps page

### ISSUE THREAD

**Issue by tfiduccia** [rancher/rancher#17730]
Version - 2.2 master 1/29

**Steps to reproduce (least amount of steps as possible):**
1.  Create a DO cluster
2.  Deploy the Longhorn catalog app

**Result:**  Pods are not showing up in the app box for longhorn.  This might be happening because the app is not fully deployed.  But as you can see, there are active pods already.

![image](https://user-images.githubusercontent.com/11514927/51947814-6628a980-23e3-11e9-80c4-1dea28b1be66.png)

![image](https://user-images.githubusercontent.com/11514927/51947698-0500d600-23e3-11e9-8852-27d171d43215.png)

**Comment by loganhz** [comment]
It will be fixed in https://github.com/rancher/helm/pull/15

**Comment by loganhz** [comment]
https://github.com/rancher/helm/pull/15 has been merged. Could you please try again?

**Comment by davidnuzik** [comment]
Version: Master 2.2 (2/1/19)
Confirmed fix. I followed the steps to reproduce and while doing so as the pods came up I saw the status bar update as they became active. And while provisioning they were shown as well.
I smoke tested in this area as well by trying with a couple more apps as well. I watched the /workloads page while watching the /apps page at the same time to ensure they are in sync and they were.
Screenshots:
All pods provisioning, shows 3 in red (not ready) state:
![screen shot 2019-02-01 at 10 01 06 am](https://user-images.githubusercontent.com/45179589/52137817-c0568400-2608-11e9-9623-598a9f981028.png)
![screen shot 2019-02-01 at 10 01 11 am](https://user-images.githubusercontent.com/45179589/52137827-c3ea0b00-2608-11e9-9863-5f2f2759413e.png)

2 of 3 are ready, status bar updates in sync:
![screen shot 2019-02-01 at 10 01 40 am](https://user-images.githubusercontent.com/45179589/52137860-db28f880-2608-11e9-830a-b2c04ddafe93.png)
![screen shot 2019-02-01 at 10 01 36 am](https://user-images.githubusercontent.com/45179589/52137865-dd8b5280-2608-11e9-88b0-9fbb3de45b13.png)
